src/services/storage/models/offer.model.ts

```diff
@@ -28,8 +27,8 @@ export default class Offer extends Model {
   @Column({ primaryKey: true, type: DataType.STRING(64) })
   provider!: string
 
-  @Column({ ...BigNumberStringType('totalCapacity') })
-  totalCapacity!: BigNumber
+  @Column
+  totalCapacity!: number
```

Comment:
Hmmm I assume this is because of the sorting right? 

Then I guess we should do some size overflow guards here... @julianmrodri do we have defined "max capacity value" for Offer that we allow or just utilizing the full types range? 

We have:
 - in contract, this is `uint128`
 - JS supports [58 bits](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER)
 - what about the DB here?

Should we maybe downsize the contract to `uint64`? With `uint64` we are supposed to get 18x10e18, so since we have the size in Mb, I would argue supporting Exa bytes would be enough? 😅

Reply:
Sqlite create san `INTEGER` I believe its 64-bit. We can downsize the contract probably just in case.

Reply:
Ok, but we are still hitting the JS limitation then...

Reply:
this bit tricky, as sequelize not support BigNumber or some lib for handling big numbers, we can not store 64-bit number using js

Reply:
The best way to resolve this will be to store it as a string and prepare a custom query with casting to number for retrieving min/max value

Reply:
Maybe we can validate the max capacity for Offers to the max number JS can handle. This can be done both in UI and contract if we want.

Reply:
@nduchak im fine with that. Would keep us safer. Is that something you can sneak at tackle?

Reply:
Yes, I guess I can tackle both parts UI and cache

Reply:
Not sure if I had enough time for that now

Reply:
@AuHau are ytou ok with custom query approach?

Reply:
Lets discuss this after daily

Reply:
Discussion notes: change this column to `BigInt` sequelize datatype and prepare a custom getter/setter for casting to`BigNumber` in JS

Reply:
Done